packages/select/src/components/query-list/queryList.tsx

```diff
@@ -152,6 +152,10 @@ export class QueryList<T> extends React.Component<IQueryListProps<T>, IQueryList
     }
 
     public scrollActiveItemIntoView() {
+        if (this.props.scrollToActiveItem === false) {
```

Comment:
turning this off _entirely_ is a no-go as it fully breaks keyboard interactions -- you can easily arrow right out of the viewport. instead, can we move this check to conditionally invoke this method in certain circumstances?

Reply:
Yeah, that's a good point, I'll investigate if there's a way to do this.